lib/Runtime/Debug/TTSnapObjects.cpp

```diff
@@ -2117,15 +2128,277 @@ namespace TTD
         }
 
 #if ENABLE_SNAPSHOT_COMPARE
-        void AssertSnapEquiv_SnapProxyInfo(const SnapObject* sobj1, const SnapObject* sobj2, TTDCompareMap& compareMap)
-        {
+        void AssertSnapEquiv_SnapProxyInfo(const SnapObject* sobj1, const SnapObject* sobj2, TTDCompareMap& compareMap) {
             const SnapProxyInfo* proxyInfo1 = SnapObjectGetAddtlInfoAs<SnapProxyInfo*, SnapObjectType::SnapProxyObject>(sobj1);
             const SnapProxyInfo* proxyInfo2 = SnapObjectGetAddtlInfoAs<SnapProxyInfo*, SnapObjectType::SnapProxyObject>(sobj2);
 
             compareMap.CheckConsistentAndAddPtrIdMapping_Special(proxyInfo1->HandlerId, proxyInfo2->HandlerId, _u("handlerId"));
             compareMap.CheckConsistentAndAddPtrIdMapping_Special(proxyInfo1->TargetId, proxyInfo2->TargetId, _u("targetId"));
         }
 #endif
+
+        //////////////////
+
+        Js::RecyclableObject* DoObjectInflation_SnapGeneratorInfo(const SnapObject* snpObject, InflateMap* inflator) {
+            Js::ScriptContext* ctx = inflator->LookupScriptContext(snpObject->SnapType->ScriptContextLogId);
+
+            SnapGeneratorInfo* generatorInfo = SnapObjectGetAddtlInfoAs<SnapGeneratorInfo*, SnapObjectType::SnapGenerator>(snpObject);
+
+            Js::Var* argVals = nullptr;
+            if (generatorInfo->arguments_count > 0) {
+                argVals = RecyclerNewArray(ctx->GetRecycler(), Field(Js::Var), generatorInfo->arguments_count);
+
+                for (Js::RegSlot i = 0; i < generatorInfo->arguments_count; i++) {
+                    argVals[i] = inflator->InflateTTDVar(generatorInfo->arguments_values[i]);
+                }
+            }
+
+            Js::ScriptFunction*  scriptFunction = nullptr;// static_cast<Js::ScriptFunction*>((generatorInfo->scriptFunction != TTD_INVALID_PTR_ID) ? inflator->LookupObject(generatorInfo->scriptFunction) : nullptr);
```

Comment:
Is this commented out code related to the "todo" further down?

Reply:
this was dead code.  I cleaned it up since we set the `scriptFunction` after object instantiation.